docs/apache-airflow/authoring-and-scheduling/deferring.rst

```diff
@@ -166,3 +166,25 @@ Airflow tries to only run triggers in one place at once, and maintains a heartbe
 This means it's possible, but unlikely, for triggers to run in multiple places at once; this is designed into the Trigger contract, however, and entirely expected. Airflow will de-duplicate events fired when a trigger is running in multiple places simultaneously, so this process should be transparent to your Operators.
 
 Note that every extra ``triggerer`` you run will result in an extra persistent connection to your database.
+
+
+Difference between Mode='reschedule' and Deferrable=True in Sensors
+-------------------------------------------------------------------
+
+In Airflow, Sensors wait for specific conditions to be met before proceeding with downstream tasks. Sensors have two options for managing idle periods: mode='reschedule' and deferrable=True.
```

Comment:
I think we should mention here that `deverrable=True` is a convention used by some operators, where `mode='reschedule'` is BaseSensor parameter. This is metioned as last point in the table, but we do not have any framework support for that and I can imagine other operators to use different convention.